crates/chronicle-synth/README.md

```diff
@@ -0,0 +1,95 @@
+# Chronicle Synth
+
+Chronicle Domain [Synth](https://www.getsynth.com/) is a program for
+generating Synth schema for Chronicle. It takes a YAML file as input,
+which contains information about the domain's name, attributes,
+entities, activities, agents, and roles, and generates Synth collections
+that can be used to generate the Chronicle operations available to a
+domain's users.
+
+If you just want to generate untyped Chronicle operations with `synth`
+the collections in `crates/chronicle-synth/synth/` are ready to go.
+
+The core functionality of Chronicle Synth is to generate the
+additional Synth schema specific to each individual Chronicle domain.
+Replace the `domain.yaml` file in `crates/chronicle-synth/` with your
+domain and run
+
+```bash
+cargo run --bin chronicle --
```

Comment:
```suggestion
cargo run --bin chronicle
```
I don't see that these trailing `--` do anything useful? Though I guess you could try,
```suggestion
cargo run --bin chronicle -- <chronicle-options> ...
```
or somesuch.